core/src/navigation/LeftNav.svelte

```diff
@@ -248,6 +253,61 @@
     });
   });
 
+  resetNavEntries = () => {
+    if(btpNavTopCnt) {
+      const moreEntries = btpNavTopCnt.querySelectorAll('.lui-moreItems  > .lui-nav-entry');
+      const navList = btpNavTopCnt.querySelector('.fd-navigation__list');
+      const spacer = btpNavTopCnt.querySelector('.fd-navigation__list > .lui-spacer');
+      moreEntries?.forEach((item) => {
+        navList.insertBefore(item, spacer);
+      });
+      btpNavTopCnt.querySelector('.fd-navigation__list > .fd-navigation__list-item--overflow').style.display = 'none';
+    }
+  }
+
+  calculateNavEntries = () => {
+    const spacer = btpNavTopCnt.querySelector('.fd-navigation__list > .lui-spacer');
+    const moreUL = btpNavTopCnt.querySelector('.lui-moreItems');
+    const entries = btpNavTopCnt.querySelectorAll('.fd-navigation__list > .lui-nav-entry');
+    if(entries.length <= 0) {
```

Comment:
would this also make sense to be less then <= 1
`  if(entries.length <= 1) {`
Because below you have   `for(let i = entries.length -1; i > 0; i--) { ` which is only looping if entries.length >=2

Reply:
unless you meant that for 1 entry present, no looping is done but rather display=flex is still set